app/views/mailing_list/steps/_name.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <%= f.govuk_error_summary %>
 
-<%= f.govuk_text_field :first_name %>
-<%= f.govuk_text_field :last_name %>
-<%= f.govuk_email_field :email %>
+<%= f.govuk_text_field :first_name, class: 'govuk-!-width-two-thirds' %>
+<%= f.govuk_text_field :last_name, class: 'govuk-!-width-two-thirds' %>
+<%= f.govuk_email_field :email, class: 'govuk-!-width-two-thirds' %>
```

Comment:
You can add the classes using built-in formbuilder functionality, https://govuk-form-builder.netlify.app/form-elements/text-field/#generating-text-inputs-with-custom-widths

Reply:
That's a nice one! I've updated it to use the `width:`